cypress/integration/page.spec.js

```diff
@@ -819,13 +876,15 @@ describe('', () => {
       });
 
       it('should not show NewCommentForm', () => {
-        newCommentForm.el().should('not.exist')
+        newCommentForm.el().should('not.exist');
       });
     });
 
     describe('after empty comments received', () => {
       beforeEach(() => {
-        cy.intercept('**/comments?postId=1', { body: [] }).as('post1CommentsRequest');
+        cy.intercept('**/comments?postId=1', { body: [] }).as(
+          'post1CommentsRequest',
+        );
         page.postButton(0).click();
         page.waitForRequest('@post1CommentsRequest');
```

Comment:
The method `page.waitForRequest('@post1CommentsRequest')` should be `cy.wait('@post1CommentsRequest')`. `waitForRequest` is not a valid Cypress command.